.github/workflows/wait_for_required_workflows.yml

```diff
@@ -51,7 +52,8 @@ jobs:
 
             // Most modules should have a 'validate-release' job
             for (const action of actions) {
-              actions = [...actions, 'validate-release']
+              // add 'validate-release / validate-release' as we use reusable with same job id
```

Comment:
I believe this needs to be in a condition and only use `'validate-release / validate-release'` if it's the AWS plugin and use `'validate-release'` for others, WDYT?

Reply:
good catch!